laundry-management-service/api/src/main/java/com/microservices/laundrymanagementapi/shema/laudry_state.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+option java_package = "com.microservices.laundrymanagementapi.messages";
+option java_outer_classname = "LaudryStateProto";
+
+message LaudryState{
+    int32 orderId = 1;
```

Comment:
Laundry state definitely should not include orderId

Reply:
fix